README.md

```diff
@@ -40,3 +47,28 @@ You can also...
  - [Discuss the documentation from a contributor perspective in our developer Slack](slack.opendatakit.org). (Use the #docs-code channel.)
  - [File an issue](https://github.com/opendatakit/docs/issues) for any needed improvements.
  - [Watch](https://github.com/opendatakit/docs/subscription) and star this repo, to keep up with what we're doing.
+
+## Troubleshooting
+
+### **1. incorrect sphinx version**
+#### Error message:
+```bash
+$ make odk1
+...
+Extension error:
+Could not import extension sphinx_tabs.tabs (exception: No module named 'sphinx_tabs')
+make: *** [odk1] Error 1
+```
+or:
+```bash
+$ make odk1
+...
+Configuration error:
+There is a programable error in your configuration file:
+...
+from sphinx.util.compat import Directive
+ImportError: cannot import name Directive
+```
+
+#### Reason & Fix:
+It caused by an incorrect` sphinx` version, and you can fix this by reinstalling `sphinx` using `requirements.txt`.
```

Comment:
It *is* caused...

How about instead of "using `requirements.txt`", "using `pip install -r requirements.txt`" so it's even easier.